firestore-send-email/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Version 0.1.8
+
+fixed - Removed code coverage check on ci
+
```

Comment:
@jhuleatt 
If we remove the ci change, then there are no changes. How do we want to represent that here?